README.md

```diff
@@ -93,12 +93,12 @@ You can as well write and read strings:
 ```csharp
 string toSend = "I ❤ nanoFramework";
 port.WriteLine(toSend);
-// this will send the string encoded finishing by a new line, by default \r\n
-// You can change the new line by anything:
+// this will send the string encoded finishing by a new line, by default `\n`
+// You can change the new line to be anything:
 port.NewLine = "❤❤";
-// Now it will send the 2 hearts as the end of line while operating a ReadLine or WriteLine
-// You can ad anytime change it back:
-port.NewLine = SerialPort.DefaultNewLine; // default is "\r\n"
+// Now it will send use 2 hearts as the line ending for `ReadLine` or `WriteLine`
```

Comment:
```suggestion
// Now it will send 2 hearts as the line ending `WriteLine` and will use 2 hearts as the terminator for `ReadLine`.
```

Reply:
I dont want to get caught up on this, but this needs a unit test, as from what I saw in source, it was a single char (rather than an array), so would not work on real hardware!